assets/models/system/codeLlama-70b-Instruct-hf/description.md

```diff
@@ -0,0 +1,118 @@
+# **Code Llama**
+Code Llama is a collection of pretrained and fine-tuned generative text models ranging in scale from 7 billion to 70 billion parameters. This is the repository for the 70B instruct-tuned version in the Hugging Face Transformers format. This model is designed for general code synthesis and understanding. Links to other models can be found in the index at the bottom.
```

Comment:
Update model card with structure as discussed in [word](https://microsoftapc-my.sharepoint.com/:w:/g/personal/ayushmishra_microsoft_com/ETjbEy6poUhBsdysFKduH3IBmrdEe2gP2Yzza3p6COAAhA?e=bstYHo)